Author: <kees@ubuntu.com>
Description: Fix FTBFS: ./sshow.c:226: error: 'CLK_TCK' undeclared.
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/sshow.c
+++ b/sshow.c
@@ -217,6 +217,7 @@
 {
 	clock_t delay;
 	int payload;
+    long CLK_TCK= sysconf(_SC_CLK_TCK);
 
 	delay = add_history(session, 0, cipher_size, plain_range);
 
@@ -265,6 +266,7 @@
 	clock_t delay;
 	int skip;
 	range string_range;
+    long CLK_TCK= sysconf(_SC_CLK_TCK);
 	
 	delay = add_history(session, 1, cipher_size, plain_range);
 	
